rust-runtime/aws-smithy-runtime/src/client/orchestrator.rs

```diff
@@ -138,12 +159,15 @@ async fn try_op(
     stop_point: StopPoint,
 ) {
     // Before serialization
+    trace!("running read_before_serialization interceptors");
     halt_on_err!([ctx] => interceptors.read_before_serialization(ctx, cfg));
+    trace!("running modify_before_serialization interceptors");
     halt_on_err!([ctx] => interceptors.modify_before_serialization(ctx, cfg));
```

Comment:
Consider calling `trace` from the `Interceptors` thing. That way, there's less lines in the invoke methods.

Reply:
Good idea...